src/backend/distributed/shared_library_init.c

```diff
@@ -361,6 +407,18 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
+	DefineCustomRealVariable(
+		"citus.distributed_deadlock_detection_factor",
+		gettext_noop("Sets the time to wait before checking for distributed "
+					 "deadlocks. The value is multiplied with Postgres' "
+					 "deadlock_timeout setting."),
```

Comment:
Isn't it the other way round? I know that multiplication is commutative here, but still, deadlock_timeout is the setting with the unit...